examples/demo-dts/package.json

```diff
@@ -88,6 +88,9 @@
   },
   "resolutions": {
     "@types/express": "4.17.21",
-    "express": "4.21.0"
+    "express": "4.21.0",
+    "@2060.io/service-agent-client": "1.1.0-dev.41",
+    "@2060.io/service-agent-model": "1.1.0-dev.41",
+    "@2060.io/service-agent-nestjs-client": "1.1.0-dev.41"
```

Comment:
I guess these resolutions are not needed... if we did everything correctly in terms of dependency management across service-agent components